### ISSUE TITLE
After installation password not working

### ISSUE THREAD

**Issue by Githubbly** [dnschneid/crouton#2422]
I am very new to working on software or, in general, tech. Just trying to get crouton installed so I can learn my first programming language in python.

I last posted on this forum a couple days ago after a problem during installation. Somehow I was able to get around it. After entering the command to install, I'm asked for a 'root password'.

![screenshot 2016-02-05 at 3 39 32 pm](https://cloud.githubusercontent.com/assets/17049618/12839785/06f3f5f0-cc20-11e5-8d7e-626ac737e17d.png)

At the time, I hadn't made any password and wasn't sure if I was supposed to make one. I just hit enter a few times with 'verify failure' appearing after each time. Then suddenly I'm asked to choose a passphrase and enter one in (after realizing that it doesn't appear on the terminal). Finally, the installation takes off.

After installation I'm asked to set a UNIX username and pw. Then I enter in the command to enter chroot which then prompts me for my pw. I use both my UNIX pw and the passphrase which I set before installation, but neither works.

![screenshot 2016-02-05 at 3 39 56 pm](https://cloud.githubusercontent.com/assets/17049618/12839791/13e9904e-cc20-11e5-9657-a8f52048f853.png)

Please explain in layman terms how I might get around this issue. Thank you in advance.

**Comment by DennisLfromGA** [comment]
When you use the '-e' option with crouton it prompts you for a password to *encrypt* your chroot, this will also set a 'chronos' password on your Chromebook. 
You don't *need* to use the 'e' option with crouton unless you want an encrypted chroot for security purposes.

The username and password requested near the end of the crouton installation are for logging into your chroot with your preferred username (and password).
